src/components/Error/Error.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  errorType: ErrorType,
+  hasError: boolean,
+  onNotificationClose: () => void,
+};
+
+export const Error: React.FC<Props> = ({
+  errorType,
+  hasError,
+  onNotificationClose,
+}) => {
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    switch (errorType) {
+      case ErrorType.Add:
+      case ErrorType.Delete:
+      case ErrorType.Update:
+      case ErrorType.Load:
+        setErrorMessage(`Unable to ${errorType} a todo`);
+        break;
+
+      case ErrorType.EmptyTitle:
+        setErrorMessage('Title can\'t be empty');
+        break;
+
+      default:
+        setErrorMessage('');
+        break;
+    }
+  }, [errorType]);
+
+  useEffect(() => {
+    const timer = setTimeout(() => onNotificationClose(), 3000);
+
+    return () => clearTimeout(timer);
+  });
```

Comment:
The useEffect should have an empty dependency array to mimic componentDidMount behavior, otherwise, the timer might be created multiple times.